### PR TITLE
Update polkadot-v0.9.29

### PR DESCRIPTION
**Pull request by OverOrion**

### CODE REVIEW

**Review by haerdib — APPROVED** [review]

**Comment by haerdib** [comment]
Ah, sorry, I forgot to check for the scripts: Can you update the launch scripts (e.g. zombienet and polkadot-launch) to v0.9.29 as well? Can be easily done via search and replace.

**Review by clangenb — APPROVED** [review]
No, it is fine for me, we should only test the runtime upgrade, when we actually perform one.

**Review by clangenb — CHANGES_REQUESTED** [review]
One minor issue. Please bump the spec version of the shell-runtime as well. :)

**Review by clangenb — COMMENTED** [review]
Also bump the shell-runtime `spec_version` here please: https://github.com/integritee-network/parachain/blob/53ba430bff5797760b21b064b7d6f925fdbef2a5/polkadot-parachains/shell-runtime/src/lib.rs#L79

File: zombienet/launch-rococo-local-with-integritee-xcm.toml [hunk 1]
```diff
@@ -42,7 +42,6 @@ name = "integritee-collator02"
 command = "./target/release/integritee-collator"
 ws_port = 9995
 
-#this is not working !!!
```
Comment (clangenb) [inline zombienet/launch-rococo-local-with-integritee-xcm.toml:45]: 🥳

**Review by clangenb — APPROVED** [review]
Nice, looks good to me!

**Pull request merged**